ci/lint/05_before_script.sh

```diff
@@ -6,4 +6,5 @@
 
 export LC_ALL=C
 
-git fetch
+git fetch origin
+git branch $CIRRUS_BASE_BRANCH origin/$CIRRUS_BASE_BRANCH
```

Comment:
needs to be guarded by ` if [ -n "$CIRRUS_PR" ]; then`

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/20697#issuecomment-748600578).